components/hab/src/command/pkg.rs

```diff
@@ -571,3 +571,33 @@ pub mod verify {
         Ok(())
     }
 }
+
+pub mod header {
+    use std::path::Path;
+
+    use common::ui::UI;
+    use hcore::crypto::artifact;
+    use std::io::{self, Write};
+
+    use error::Result;
+
+    pub fn start(ui: &mut UI, src: &Path) -> Result<()> {
+        try!(ui.begin(format!("Reading Artifact Header for {}", &src.display())));
```

Comment:
I would recommend not capitalizing all this, and also, we have in the past decided not to refer to `.hart`s as artifacts, just as packages. So altogether:

`Reading package header for {}`